src/app/harbor/shipyard/new-ship-form.tsx

```diff
@@ -157,6 +157,28 @@ export default function NewShipForm({
       return
     }
 
+    if (!screenshotUrl.startsWith('http')) {
```

Comment:
Won't this fire for `https` links too? `https://` starts with `http`!

Reply:
should both http and https links be allowed or just http?

Reply:
Just https.